<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LegacyCameraSettingsStreamsPanel</class>
 <widget class="QWidget" name="LegacyCameraSettingsStreamsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>207</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <item>
    <widget class="nx::vms::client::desktop::InputField" name="cameraIdInputField" native="true"/>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::InputField" name="primaryStreamUrlInputField" native="true"/>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::InputField" name="secondaryStreamUrlInputField" native="true"/>
   </item>
   <item>
    <widget class="QPushButton" name="editStreamsButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Edit Streams...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::InputField</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/input_field.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
